source/customer_api.rst

```diff
@@ -495,3 +495,184 @@ E. Legacy API
 Legacy endpoint's documentation is available at `Legacy Membership API <http://docs.stamps.co.id/en/latest/legacy_customer_api.html>`_
 
 
+
+5. Add Child
+===============
+| URL endpoint: https://stamps.co.id/api/v2/memberships/add-child
+| Allowed Method: POST
+| Require Authentication: Yes
+
+A. Request
+-----------------------------
+
+You can use this API to update your customer's profile through Point of Sales
+or other websites.
+
+============= =========== =========================
+Parameter     Required    Description
+============= =========== =========================
+user          Yes         Customer's integer primary key or Card number
+token         Yes         Authentication string
+name          Yes         Child's name
+birthday      Yes         Child's birthday (with format YYYY-MM-DD)
+gender        Yes         Child's gender ("m" or "f")
```

Comment:
Yang udah self descriptive nggak perlu repeat `Child's birthday` lagi. Langsung `In YYYY-MM-DD format` aja.